Task4.js

```diff
@@ -33,10 +33,11 @@ console.log(fruits(10, 10));
 // -В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
 
 function terminal(name) {
+    let newName = name.slice(0, 1) + name.slice(1, name.length).toLowerCase();
     let arrNames = ['Ksenia', 'Masha', 'Dasha', 'Irina'];
```

Comment:
As option you need to stor all names in lower case and transform parameter `name` value to lower case. And then you need to equal them.